i18n/en.po

```diff
@@ -24,11 +24,11 @@ msgstr ""
 "mesh network over mesh VPN and share your internet connection until the key has been "
 "registered on the Freifunk Magdeburg servers. To register the key enter it together "
 "with your node's name (<em><%=escape(hostname)%></em>) and some more information at "
-"<a href=\"http://register.md.freifunk.net/\">http://register.md.freifunk.net/</a> or "
-"directly use this link to <a href=\"http://register.md.freifunk.net/#new?hostname=<"
-"%=urlescape(hostname)%>&key=<%=pubkey%>&mac=<%= sysconfig.primary_mac %>\">register "
-"your node <em><%=escape(hostname)%></em></a>.</p><div class=\"the-key\"># <%= "
-"escape(hostname) %><br/><%= pubkey %></div>"
+"<a href=\"https://register.md.freifunk.net/\">https://register.md.freifunk.net/</a> "
+"or directly use this link to <a href=\"https://register.md.freifunk.net/#new?"
```

Comment:
Dass hier auch Dinge wie "or" und "hostname" geändert wurden, liegt wahrscheinlich daran, dass der PO-Editor die Linebreaks verschiebt, oder?

Finde ich nicht so schön (die Semantik der Änderung geht in der Anpassung des Markups verloren), lässt sich aber wohl nicht vermeiden.

Reply:
Genau so ist es, Poedit setzt die Linebreaks ggf. neu. Man könnte die .po-Dateien auch mit einem Texteditor bearbeiten, aber ich benutz das lieber Poedit, weil ich mir dann nicht versehentlich das Dateiformat kaputt mache. Geändert wurde hier tatsächlich nur _http_ zu _https_. Alles andere sind Änderungen von Poedit.